src/BuiltinExtensions/ComfyUIBackend/ExtraNodes/SwarmComfyExtra/SwarmYolo.py

```diff
@@ -40,22 +43,46 @@ def seg(self, image, model_name, index):
         else:
             masks = masks.data.cpu()
         masks = torch.nn.functional.interpolate(masks.unsqueeze(1), size=(image.shape[1], image.shape[2]), mode="bilinear").squeeze(1)
+
+        sortedindices = self.sort_masks(masks, sort_order)
```

Comment:
this is in the wrong place, it was in the correct place previously, please move it back

Reply:
Where it was before your PR is the correct place for it to be. Index 0 and index longer than mask count do not require sorting.

Reply:
Ah, yes, that makes sense. Thank you. I'll fix this.